features/show/tabs.feature

```diff
@@ -23,5 +24,8 @@ Feature: Show - Tabs
       end
     """
     Then I should see two tabs "Overview" and "Details"
-    And I should see "tab 1"
-    And I should see "tab 2"
+    And I should see the element "#overview span"
+    And I should not see the element "#details span"
+    Then I follow "Details"
+    And I should not see the element "#overview span"
+    And I should see the element "#details span"
```

Comment:
@deivid-rodriguez , suddenly when I use
```And I should see "tab 1"``` it always return true  because it just check content of page (no matter visible or not) so I changed it to css selectors to make it at least display difference between tabs. 
and yes it doesn't work without js (do not change visibility of content)